src/tools/kdb/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ if (BUILD_SHARED)
 	add_executable (kdb $<TARGET_OBJECTS:kdb-objects>)
 	add_dependencies (kdb kdberrors_generated elektra_error_codes_generated)
 
-	target_link_libraries (kdb elektra-core elektra-kdb elektratools)
+	target_link_libraries (kdb elektra-core elektra-kdb elektratools elektra-proposal)
```

Comment:
This is ugly and should be avoided. Which methods from proposal do you need?

Reply:
I think, this is actually just an artifact from some earlier tests, because I didn't actually change the `kdb` tool. I just used it for testing.

Reply:
Ok, perfect, then you can simply change it back.